Sources/Testing/Testing.docc/TemporaryGettingStarted.md

```diff
@@ -144,3 +144,35 @@ using Xcode 16 Beta.
 See the [Test Explorer](https://www.swift.org/documentation/articles/getting-started-with-vscode-swift.html#test-explorer)
 section of
 [Getting Started with Swift in VS Code](https://www.swift.org/documentation/articles/getting-started-with-vscode-swift.html).
+
+#### Running tests for WebAssembly
+
+To run tests for WebAssembly, install a Swift SDK for WebAssembly by following
+[these instructions](https://book.swiftwasm.org/getting-started/setup-snapshot.html).
+
+Because `swift test` doesn't know what WebAssembly environment you'd like to use
+to run your tests, building tests and running them are two separate steps. To
+build tests for WebAssembly, use the following command:
+
+```sh
+swift build --swift-sdk wasm32-unknown-wasi --build-tests
+```
+
+After building tests, you can run them using a [WASI](https://wasi.dev/)-compliant
+WebAssembly runtime such as [Wasmtime](https://wasmtime.dev/) or
+[WasmKit](https://github.com/swiftwasm/WasmKit). For example, to run tests using
+Wasmtime, use the following command (replace `{YOURPACKAGE}` with your package's
+name):
+
+```sh
+wasmtime .build/debug/{YOURPACKAGE}PackageTests.wasm
+```
+
+Most WebAssembly runtimes forward trailing arguments to the WebAssembly program,
+so you can pass command-line options of the testing library. For example, to list
+all tests and filter them by name, use the following commands:
+
+```sh
+wasmtime .build/debug/{YOURPACKAGE}PackageTests.wasm --list-tests
```

Comment:
```suggestion
wasmtime .build/debug/{YOURPACKAGE}PackageTests.wasm list --testing-library swift-testing
```

Reply:
(I'll be adding support for the `test` subcommand in place of `--list-tests` momentarily.)

Reply:
@grynspan `wasmtime .build/debug/{YOURPACKAGE}PackageTests.wasm` is just equivalent to `.build/debug/{YOURPACKAGE}PackageTests.xctest`. Since `list` is an argument of SwiftPM, it doesn't exist in `{YOURPACKAGE}PackageTests.wasm` just as it doesn't in `{YOURPACKAGE}PackageTests.xctest`, does it?

Reply:
#589 allows you to just say `list`.

Reply:
That makes sense. Thank you for letting me know.